src/cljs/project_scoper/components/bubble.cljs

```diff
@@ -1,16 +1,27 @@
 (ns project-scoper.components.bubble
-  (:require [re-frame.core :as rf]))
+  (:require [re-frame.core :as rf]
+            [reagent.core :as reagent]
+            [project-scoper.events :as events]
+            [project-scoper.subs :as subs]
+            [project-scoper.helpers :refer [keywordize-name]]))
 
 (def colors {:red "#EB5757"
              :green "#6FCF97"
              :blue "#56CCF2"
              :purple "#BB6BD9"
              :yellow "#F2C94C"})
 
-(defn bubble [color-key text & [event]]
-  (let [color-val (some #(when (= color-key (first %)) (second %)) colors)]
-    ; TODO: add selection functionality
-    [:div.bubble {:style {:border-color color-val
-                          :color color-val}
-                  :on-click #(rf/dispatch event)}
-      [:div.text text]]))
+(defn bubble [color-key text db-key]
+  (let [color-val (color-key colors)]
+    (fn []
+      (let [db-value (keywordize-name text)
```

Comment:
can we combine the two let statements into one ? we can move `db-value (keywordize-name text)` and ` selected?..` binding into the first `let`